css/style.css

```diff
@@ -0,0 +1,78 @@
+* {
+	margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+  margin: 0px;
+  background-color: #83A298;
+  padding: 0px;
+  min-height: 100%;
+  width: 100%;
+}
+
+/* Style query form */
+.queryform {
+    overflow: hidden;
+    background-color: #BCCDC7;
+    margin: auto;
+    padding: 10px;
+    margin: 20px 20px 0px 20px;;
+}
+
+/* Style the past search links */
+.history a {
+    float: left;
+    display: block;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+/* Change color on hover */
+.history a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+.content {
+	background-color: #83A298;
+}
+
+/* Style the info */
+.info {
+    background-color: #BCCDC7;
+    padding: 10px;
+    margin: 0px 20px 0px 20px;
```

Comment:
Can be simplified to `margin: 0 20px;`. This is also seen elsewhere. Here's a helpful table.

| CSS Entry   | Description                                                                           |
|-------------------|--------------------------------------------------------------------------------------------------------------------------------|
| `margin: a;`       | Margin of size `a` will apply in all four directions                                                                           |
| `margin: a b;`     | Margin of size `a` will apply to top and bottom directions, margin of size `b` will apply to left and right directions         |
| `margin: a b c;`   | Margin of size `a` will apply to top, margin of size `b` will apply to left and right, margin of size `c` will apply to bottom |
| `margin: a b c d;` | Top margin of size `a`, right margin of size `b`, bottom margin of size `c`, left margin of size `d`                           |